osu.Game.Rulesets.Osu/Difficulty/OsuPerformanceCalculator.cs

```diff
@@ -43,8 +43,8 @@ protected override PerformanceAttributes CreatePerformanceAttributes(ScoreInfo s
 
             double multiplier = PERFORMANCE_BASE_MULTIPLIER;
 
-            if (score.Mods.Any(m => m is OsuModNoFail))
-                multiplier *= Math.Max(0.90, 1.0 - 0.02 * effectiveMissCount);
+            //if (score.Mods.Any(m => m is OsuModNoFail))
```

Comment:
Don't put random changes in unrelated PRs. And you have to explain why it's "pointless" too before removing it